packages/Chem/src/widgets/properties.ts

```diff
@@ -56,6 +85,70 @@ export function getMoleculeCharge(mol: OCL.Molecule): number {
   return moleculeCharge;
 }
 
+export async function statsWidget(molCol: DG.Column<string>): Promise<DG.Widget> {
+  const host = ui.div();
+  host.style.marginLeft = '-25px';
+  const size = Math.min(molCol.length, 1000);
+  const randomIndexes = Array.from({length: size}, () => Math.floor(Math.random() * molCol.length));
```

Comment:
Are we doing random sampling here? We don't mention it anywhere in the UI

Reply:
This method is not good since you can end up using multiple values twice. Think how you can make it better (for instance, the way we do it in Detector.sampleCategories)